### PR TITLE
Fixed rendering Equation in Dark Mode problem

### PR DESCRIPTION
**Pull request by Mohitkumar6122**
Fixes to #5.
Currently in Dark Mode equation was not visible so instead of displaying image using image tag, I uploaded the image and then used it in the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ A Tier 3<sup>[1](#footnote1)</sup> method is used for estimation of aboveground
 
 ### Eqn. 1 Chapman-Richards Function
 
-<img src="https://render.githubusercontent.com/render/math?math=AGC=max\times[1-e^{-k\cdot age}]^{(1/1-m)}">
+![Equtaion](https://user-images.githubusercontent.com/47894634/121567047-bdbb9300-ca3b-11eb-861e-4605ade57ee2.png)
```
Comment (aornugent) [inline README.md:7]: Change to `[Equation]`

**Comment by aornugent** [comment]
Hi @Mohitkumar6122 - thanks for the contribution!

I think your image has some extra borders when displayed in light mode:

image

Could you please re-crop it to work in both themes?

**Comment by Mohitkumar6122** [comment]
> Hi @Mohitkumar6122 - thanks for the contribution!
> 
> I think your image has some extra borders when displayed in light mode:
> 
> image
> 
> Could you please re-crop it to work in both themes?

Sure @aornugent,I will do it ASAP.

**Comment by Mohitkumar6122** [comment]
@aornugent , I have set up another PR for this change #7 .

**Pull request closed without merging**